src/components/portrait/portrait-initials.component.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Browser from '../../utils/helpers/browser';
+
+class PortraitInitials extends React.Component {
+  static propTypes = {
+    /** A custom CSS class for the component. */
```

Comment:
Doesn't really matter but one line comments could just be `//`. There seems to be a mix in this file.

Also wondering if the propTypes need, or if they are self explanatory?